app/Http/routes.php

```diff
@@ -35,7 +35,7 @@
 // Nomination
 Route::post('nomination', ['as' => 'nomination.create', 'uses' => 'NominationController@store']);
 // Admin
-Route::group(['before' => 'role:administrator', 'prefix' => 'admin'], function () {
```

Comment:
Hmm, yeah I guess that middleware got lost! It'd be nice to have a generic "role" middleware if there are more than one type of role that you would want to limit access by. Otherwise this is good!